examples/volumes/glusterfs/README.md

```diff
@@ -84,17 +87,13 @@ You can verify that the pod is running:
 $ kubectl get pods
 NAME             READY     STATUS    RESTARTS   AGE
 glusterfs        1/1       Running   0          3m
-
-$ kubectl get pods glusterfs -t '{{.status.hostIP}}{{"\n"}}'
-10.240.169.172
 ```
 
-You may ssh to the host (the hostIP) and run 'mount' to see if the Glusterfs volume is mounted,
+You may execute the command `mount` inside the container to see if the GlusterFS volume is mounted correctly:
```

Comment:
Yes, what you say makes sense.

I think the correction is still an improvement, because with this command you can still verify that the container has the GlusterFS volume mounted correctly, it's easier to execute with `kubectl exec` instead of ssh'ing to the node, and it's easier to see if multiple container mount a gluster vol on this host.